pkg/controller/migration/controller.go

```diff
@@ -217,3 +227,55 @@ func (r *Reconciler) reflectPlan(plan *api.Plan, migration *api.Migration) {
 	}
 	migration.Status.VMs = plan.Status.Migration.VMs
 }
+
+// Set owner reference for Migration.
+// This is needed when the CR is created with CLI and not through UI which automatically
```

Comment:
```suggestion
// This is needed so the migration CR will be auto deleted once the plan CR is deleted
```

hi,
Once this is merged the UI (and any other automation) can remove the code that adds the owner reference. once they do this, this comment will become obsolete, WDYT?

Reply:
Hey, good point, I will change it